flocker/provision/_install.py

```diff
@@ -505,7 +520,7 @@ def task_open_control_firewall(distribution):
     """
     if distribution in ('centos-7', 'fedora-20'):
         open_firewall = open_firewalld
-    elif distribution == 'ubuntu-14.04':
+    elif distribution in ('ubuntu-14.04', 'ubuntu-15.04'):
```

Comment:
I'm not sure about that. What if Ubuntu 16.04 has a breaking change? And some things will have to change I think because 15.04 uses SystemD instead of Upstart by default.

The reason I think that they should be reverted is because it leaves the code in the state of part-supporting 15.04. Better to do it all in one go. It might be more OK to leave this if we had very short term plans to support 15.04 on the node.